migration/src/main/resources/migrations/ensure-orders-have-encounters.xml

```diff
@@ -17,28 +17,43 @@
            then create the encounters for orders that don’t have encounters
         -->
         <sql>
-            There are over 300,000 orders on both Rwink and Butaro with no linked encounter
-            select count(*) from orders where encounter_id is null
+            SET @EncTUUID :=  "a9ef19d1-8875-11ea-8c74-7a7919290ad6";
 
-            Of these, there are still between 100k and 150k on each server with no encounters on the same day
 
-            select count(*)
-            from orders o
-            left join encounter e on o.patient_id = e.patient_id and date(o.start_date) = date(e.encounter_datetime)
-            where e.encounter_id is null;
+            /* Create EncounterType */
+            INSERT INTO `encounter_type`
+            (`name`,`description`,`creator`,`date_created`,`uuid`)
+            VALUES
+            (
```

Comment:
Can we format this consistently?  Eg. always put the parenthesis on the same line, or always on different lines?

Reply:
Done.